pkg/sbproxy/reconcile/reconcile_visibilities.go

```diff
@@ -139,8 +139,9 @@ func (r *resyncJob) getVisibilitiesFromSM(ctx context.Context, smPlansMap map[br
 
 func (r *resyncJob) convertSMVisibility(visibility *types.Visibility, smPlan *types.ServicePlan, broker *platform.ServiceBroker) []*platform.Visibility {
 	scopeLabelKey := r.platformClient.Visibility().VisibilityScopeLabelKey()
+	shouldBePublic := visibility.PlatformID == "" || len(visibility.Labels[scopeLabelKey]) == 0
```

Comment:
shouldn't `scopeLabelKey` be checked if empty still?

Reply:
`len(visibility.Labels[scopeLabelKey]) == 0` covers this case - doesn't it?
 If the `scopeLabelKey` is empty, then there won't be any value in the labels with the key empty string.